src/speechclient.hh

```diff
@@ -1,6 +1,6 @@
 #pragma once
 #ifdef TTS_SUPPORT
-
+  #include "fixx11h.h"
```

Comment:
The current code have x11 headers in headers.

In moc compiled code, it will pollute other headers.

It was never a problem, because the header that includes x11 headers ranked lower.

But after this change, it ranks differently. Hence this thing.

Needs another time to properly move all x11 code out of header into `.cc`.